### DISCUSSION TITLE
Аfter downloading, the map goes into the open collection

### DISCUSSION THREAD

**Discussion by Mackaronn** [ppy/osu#30725] [Q&A]
In Laser, if you download a map, then go to the collection you created and then click on the downloaded map in notifications, then this map will appear in this folder, although it should not be in it.

https://github.com/user-attachments/assets/67ab6a9a-70c5-4513-93c8-e1ab87db5e0e

**Comment by peppy** [comment]
It will appear until you select another track. This is intended else it would break the game experience. Unless you want it to deselect your collection filter when clickign the notification?
